yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -774,3 +777,180 @@ func (yb *TargetYugabyteDB) recordEntryInDB(tx pgx.Tx, batch Batch, rowsAffected
 	}
 	return nil
 }
+
+func (yb *TargetYugabyteDB) GetDebeziumValueConverterSuite(snapshotMode bool) map[string]ConverterFn {
+	converterValueSuite := make(map[string]ConverterFn)
```

Comment:
Declare and populate `ybValueConverterSuite` at the top of this file. Only return the reference to `ybValueConverterSuite` map from the `GetDebeziumValueConverterSuite()`. That way, no matter how many times the function is called, only one map is created.